aspnetcore/mvc/controllers/filters/samples/8.x/FiltersSample/Controllers/ExceptionController.cs

```diff
@@ -1,13 +1,13 @@
-﻿using FiltersSample.Filters;
+using FiltersSample.Filters;
 using Microsoft.AspNetCore.Mvc;
 
 namespace FiltersSample.Controllers;
 
 // <snippet_Class>
-[TypeFilter(typeof(SampleExceptionFilter))]
+[TypeFilter<SampleExceptionFilter>]
```

Comment:
I'm assuming that based on the directory these will only surface for 8.x and above docs?

Reply:
> 

Yes, these changes only show up in 8.0 and later.